dtreeviz/trees.py

```diff
@@ -1143,9 +1148,23 @@ def _class_split_viz(node: ShadowDecTreeNode,
 
     ax.set_xlim(*overall_feature_range_wide)
 
-    wedge_ticks = _draw_wedge(ax, x=node.split(), node=node, color=colors['wedge'], is_class=True, h=h, height_range=height_range, bins=bins)
-    if highlight_node:
-        _ = _draw_wedge(ax, x=X[node.feature()], node=node, color=colors['highlight'], is_class=True, h=h, height_range=height_range, bins=bins)
+    if node.is_categorical_split() and type(X_feature[0]) == str:
+        # run draw to refresh the figure to get the xticklabels
+        plt.draw()
```

Comment:
seems like a hack. anyway to just use the right labels? i guess plot doesn't always show all labels...is that the idea?